companies/serializers.py

```
@@ -263,3 +264,42 @@ def get_logo(self, obj):
+    def get_total_revenue(self, obj):
+        """총 매출액 추출"""
+        return obj.raw_values.get('total_revenue', 0)
+
+    @extend_schema_field(OpenApiTypes.OBJECT)
+    def get_segments(self, obj):
+        """왼쪽 노드: 중앙 매출액 노드로 들어오는 모든 source 값(사업부별 매출) 추출"""
+        if not obj.nodes:
+            return []
+        # 중앙 노드(매출액/영업수익)의 이름을 가져옴
+        target_name = obj.nodes[0]['name']
+        return [{"name": l['source'], "value": l['value']} for l in obj.links if l['target'] == target_name]
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)